packages/runner/test/mocha.opts

```diff
@@ -1,3 +1,3 @@
---require '@packages/web-config/node-register'
+--require ../web-config/node-register
```

Comment:
It technically does work as it was originally written. The reason I changed it was because, like node_modules, a package is an opaque data structure; meaning we should not reach into it directly to pull out files. This is why the main field exists.

**Unfortunately**, there are many packages in the wild that promote reaching directly into the package, so it isn't exactly an uncommon practice. Although I do not feel we should violate the above principle of packages and node_modules in general, I'm not super passionate about this change, so I can change it back if it is felt we should continue that practice.